content/en/docs/instrumentation/erlang/resources.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Resources
+weight: 36
+---
+
+## Resources
+
+A [resource](/docs/reference/specification/overview/#resources) represents the
+entity producing telemetry as resource attributes. For example, an OTP Release
+producing telemetry that is running in a container on Kubernetes has an OTP
+Release name, a Pod name, a namespace, and possibly a deployment name. All four
+of these attributes can be included in the resource.
+
+In your observability backend, you can use resource information to better
+investigate interesting behavior. For example, if your trace or metrics data
+indicate latency in your system, you can narrow it down to a specific container,
+pod, or Kubernetes deployment.
+
+### Using resource detectors
+
+Resource detectors fetch resource attributes from various sources. The default
+detectors use the OS environment variable `OTEL_RESOURCE_ATTRIBUTES` and the
+`opentelemetry` OTP Application environment variable `resource`.
+
+The detectors to use is a list of module names and can be configured in the
+Application configuration:
+
+<!-- prettier-ignore-start -->
+{{< tabpane langEqualsHeader=true >}}
+
+{{< tab Erlang >}}
+%% sys.config
+{resource_detectors, [otel_resource_env_var, otel_resource_app_env]}
+{{< /tab >}}
+
+{{< tab Elixir >}}
+## runtime.exs
+resource_detectors: [:otel_resource_env_var, :otel_resource_app_env]
+{{< /tab >}}
+
+{{< /tabpane >}}
+<!-- prettier-ignore-end -->
+
+Or through the environment variable `OTEL_RESOURCE_DETECTORS`:
+
+```sh
+OTEL_RESOURCE_DETECTORS=otel_resource_env_var,otel_resource_app_env
+```
+
+Since resource detectors may hit the network there is a timeout that when
```

Comment:
Maybe this will read a little bit better:

All resources detectors are protected with a timeout after which they return an empty value. This allows you to use a resource detector that fetches attributes over the network. The default timeout is 5 seconds and can be set with with environment variable `OTEL_RESOURCE_DETECTOR_TIMEOUT` or Application variable `otel_resource_detector_timeout`.

Reply:
I like it. I've pushed a change that mostly takes this paragraph.